Q1-2024 Sven Template Updates.md

sven-pa opened the pull request.

sven-pa committed:
Update SWDP: removed outdated placeholder.md

sven-pa committed:
Update 13485-mapping for KPIs/annual goals.md

sven-pa committed:
DRC-SOP: clarified checking periods.md

- Clarified wording to apply A/B/C logic of categories
- Changed wording from reviewing periods to checking periods

sven-pa committed:
Updated SOP Vigilance for MIR form.md

Based on customer auditing feedback, added a reference to the European Commission's Manufacturer Incident Report (MIR).

sven-pa committed:
IFU - Added reporting to authorities by users.md

sven-pa committed:
GSPR Checklist - Removed 14.2.b as applicable.md

Following client TD review findings

sven-pa committed:
GSPR checklist - ,arked 17.2 as not applicable.md

Following client TD review findings.
Only applicable for mobile applications, not all software.

sven-pa committed:
Updated GSPR checklist - new 15223 version.md

Following client TD review findings.
Replaced 2017 by 2021 version reference.

sven-pa committed:
ISD-SOP: added IU plus formatting fixes.md

sven-pa committed:
ISD-SOP: removed UEP as input in step 3 (output only).md

ivoflipse commented on templates/general/checklist-mdr.md:
```diff
@@ -148,7 +148,7 @@
 | 17.1 | Devices that incorporate electronic programmable systems, including software, or software that are devices in themselves, shall be designed to ensure repeatability, reliability and performance in line with their intended use. In the event of a single fault condition, appropriate means shall be adopted to eliminate or reduce as far as possible consequent risks or impairment of performance ...
 | 17.2 | For devices that incorporate software or for software that are devices in themselves, the software shall be developed and manufactured in accordance with the state of the art taking into account the principles of development life cycle, risk management, including information security, verification and validation. | Yes |  | IEC 62304:2006 / AMD1:2015 | Software Development and Maintenanc ...
-| 17.3 | Software referred to in this Section that is intended to be used in combination with mobile computing platforms shall be designed and manufactured taking into account the specific features of the mobile platform (e.g. size and contrast ratio of the screen) and the external factors related to their use (varying environment as regards level of light or noise). | Yes |  | IEC 62304:2006 / A ...
+| 17.3 | Software referred to in this Section that is intended to be used in combination with mobile computing platforms shall be designed and manufactured taking into account the specific features of the mobile platform (e.g. size and contrast ratio of the screen) and the external factors related to their use (varying environment as regards level of light or noise). | No | Only applicable for mo ...
```
Medical Imaging products may have requirements for the environment or the screens they're viewed on. The FDA for example made quite a fuss about this for ONE Planner Hip

sven-pa replied:
Thanks for the pointer Ivo! But as this template refers to the MDR and we're trying to generally keep things minimalistic, I'd keep the wording as such for now.

sven-pa committed:
ISD-SDWP: added important specs.md

Added:
- 62304 classification
- Rationale on measurement function
- Rationale on combination device

sven-pa committed:
Layout fix.md

sven-pa committed:
Layout fix-2.md

sven-pa committed:
Layout fix-3.md

olieidel commented on templates/techdoc/62304/software-development-maintenance-plan.md:
```diff
@@ -52,8 +52,16 @@ Please see the relevant processes for the following activities:
-
-Describe your device's software safety class according to IEC 62304 and your reasoning behind the classification.
+**IEC 62304 Software Safety Classification**
+The software safety classification for [enter device name] has been established as class [XXXX] per IEC 62304:2006/AMD1:2015 based on the decision-making process outlined in table 3 and in paragraph 4.3 of the norm. A malfunction of, or latent design flaw in the software device may lead to situations with unacceptable risks [for example: false-positive and false-negative diagnosis, resulting in u ...
```
- Add a backslash `\` before bracket characters `[` `]`.
- Markdown paragraphs need an additional newline (enter) to be displayed as paragraph. Currently, the paragraphs here are only separated by one newline. Here's how it currently looks. You can check this yourself by viewing the markdown file in GitHub (if that makes sense).

![Screenshot 2024-06-01 at 13 34 18](https://github.com/openregulatory/templates/assets/12237543/3a86eb2a-d30c-4b04-be00-148270f706a6)

sven-pa replied:
Thanks - should be updated now!

sven-pa committed:
SWDP format fices.md

olieidel committed:
use markdown italic formatting instead of html em tag

olieidel committed:
add backslashes before brackets

olieidel committed:
Merge branch 'master' into sven-pa-patch-1
